activity/s3/README.md

```diff
@@ -1,6 +1,7 @@
 # Amazon S3
 
-Upload or Download files from Amazon Simple Storage Service (S3)
+Upload or Download files from Amazon Simple Storage Service (S3). The Key Id and Secret Access Key needs to be set as Enviornment variable.
```

Comment:
I would revert this, since its not always the case that it uses environment variable